### PR TITLE
Add command to change OpenJDK versions from 11 to 8

### PR DESCRIPTION
**Pull request by ryanbourdais**
Added a new job to allow switch OpenJDK versions per [this issue](https://github.com/CircleCI-Public/android-orb/issues/31)

Description
Wrote a command with 3 steps, the first to check the current OpenJDK version, the second changes the default OpenJDK version of the current image to OpenJDK v8, and the third step confirms the change by running java -version once more
Also, added a parameter that will allow you to change OpenJDK versions, default: '1.8.0'. To check available OpenJDK versions, run `update-java-alternatives --list`

### CODE REVIEW

**Review by JalexChen — CHANGES_REQUESTED** [review]
1) I would probably make the comments in a separate commit to keep the PR cleaner
2) That command is not available on newer versions of java (e.g  the default 11 - gonna need to find something different. Additionally, you may need to install the actual version of Java before you can set it
3) I'm not sure about the JAVA_HOME significance, but CircleCI's environment can be different than most; the PATH could be different
4) I would also put a test into the actual configuration to see if this works

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:7588031559f4e01ba198c69979bd5d6b64a609f2

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:dd3024e5ed6c1d4f7c30bb6aff3aefed9de3b7e8

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:d1a343d7eecb7d1d14e2e411285703fc595ad143

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:b8a007f73bdaf3cfd1fc14c519e76e097c2db469

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:c3e90ec8595813a1117fad368c50123a3fe6006c

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:6e09439e23f236c0a872b039af53a11a1d18cb86

**Comment by ryanbourdais** [comment]
Added check to ensure that the requested java version is not the currently set version,  as well as added code to set the JAVA_HOME and PATH variables following the installation of a new java version

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:2114a13d668e90a1341e54d9d3a8c30630376196

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:219709b83a9078c8ff05e2b5f854e0ea42edfba4

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:b9d95c938358a86657060745c521fdf8d6e0a4b9

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:7c6e026ed9f0bcf33c3652e689ab618e9f8946fd

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:da34a835404de6f7871141ccac225bf72a9450fe

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:8c47c72d0506c3ec4c95a4dd5a0f1ad8f5710bd5

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:35d88ad6f7cd0344718eb3088e9fb1bf356cc4f7

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:08ecdfd446b1986a2e415fdc9a61b9630dcc1247

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:227bc2a6bb78161a69f4e2416ef9121bdafb8cf6

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:03ed086c46de15bb693b2c1a29566f3b86597fb5

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:9bc6df0eac6229f1121dd3b978694c98eb7896e8

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:cbf42082c5663b155646352dd4db9629f469d950

**Review by JalexChen — APPROVED** [review]

**Comment by JalexChen** [comment]
i think it looks fine, but i would want to get a second review as well

File: src/commands/change-java-version.yml
Comment (EricRibeiro) [inline src/commands/change-java-version.yml:12]: You almost always want to wrap your subshell commands in double-quotes.  There is a very succinct explanation with examples [here](https://www.oreilly.com/library/view/linux-shell-scripting/9781785881985/f3497351-d8d0-4de2-be20-4f169b59ebe3.xhtml).

File: src/commands/change-java-version.yml
Comment (EricRibeiro) [reply src/commands/change-java-version.yml:14]: In shell, `==` is generally used for string comparison. To compare numbers you want to use `-eq`. 

You can read [this](https://stackoverflow.com/a/20449556) and [this](https://tldp.org/LDP/abs/html/comparison-ops.html#ICOMPARISON1) for more information.

File: .circleci/test-deploy.yml
Comment (EricRibeiro) [inline .circleci/test-deploy.yml:21]: Are any specific reasons behind the "1" suffix in the test job name?

**Review by EricRibeiro — CHANGES_REQUESTED** [review]
Great Work! 🥇 

You should include your new test job in the [requirement list](https://github.com/CircleCI-Public/android-orb/blob/main/.circleci/test-deploy.yml#L10) for publishing. That way, if it fails, the orb won't go to production.

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:a2d8deb0fb542bb1d3a8985a6cb3d5667a5ea1b9

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:eb34324247739284c0394a77a946291afdd86a6a

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:645bb0c29fac2d4c26fc05fa97f472bd3107f272

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:dc28f8678e80959ef741ec34e2a7ce933a9a708e

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:3c07eca5e6808d4ba426a47aefeeee4093f88ad4

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:f08a0efe1ad7d72174cf9577e8f83467cfef0dab

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:a45a377ab60f846ba6b6c5f6a6c2d0ec589388b6

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:a65afbfbfbb25e4522957d8842315e0057f891c5

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:edcf97483f0a972b1ca3f8610cd11e174a6669fb

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:93d1123dac6d25f3425b8ffe44e2653459024b20

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:dc6d568e9b4ab0b840da94d20c81fb5289b2bc1f

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:5a0a49fc79a5f92685d92a8241a2799f49edcadd

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:2abd4ac794a1bfdfac2212f6bd908aa745df6c10

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:a99868e14557f0877bc3fc282cc6a89bbfdd04dc

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:ffd5851e0dbe9e34e5d61d349039d0868bc44449

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:d4f1d6f5440ca5ab81418981cbee45939feb1e6c

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:bda7e170d489d6d31fd7a706fee23f3d33761b5c

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:eb9cb6a1140c18d1cd57923274b504ff42cc474e

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:8880256e105a4e4422a7cfbca8225de2adb59426

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:e66486a0bd3344ea52f4dcbd6c8e10b71caf2920

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:6e55c905be77132e99638971985b788b57cee05b

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:7af9a804c9a2fd14f950dc5873ccb69bbb6badd4

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:539a651dfa3261d7aace9f07fb4770333b32d4d5

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:4faa9126e5ed869ca1130d401f486141f6310f83

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:868a3f990c502c0bd71a0063731926f690d8192e

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:d0e037c283a82c797704a7bbc2b83c993dfea35b

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:6a1baf326646539568cb35ebb7f6e5269176191d

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:743a7fceca2a9e261e8a5d04849da642838036b1

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:bfd3fa181b3fe1e996d6a5b5831f3377c9c461b9

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:13c4a903038b129ea99d8ef9c838270f21fadae8

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:aa6ebfc48cb76a6a4abf8f6d639068c722ba7575

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:abe7ea6202eab654f78f3f7412467fb1a4b73317

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:915a662c05b8b749b3fa58cbc73bb5bb5f73288d

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:bab1abf6ebbeadd174430e2f3ac70a5f6367cd30

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:46fc9291528b4446d64ae10ea3ab2a7d975b166a

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:9ee9d15a2e1b38f2579b347d1b1eb0a2e4bd7ad1

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:80040d1c2f2e96849bdb315351d39c08ef6c4436

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:86c23db5d8db2a33c6354e9e7733c268f8f18ce8

**Review by EricRibeiro — APPROVED** [review]
LGTM

**Review by felicianotech — DISMISSED** [review]

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:86f3c382b8a94e741ac01e3f92a46eb9c1f5440e

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:737ab191652cc89837a23575f822d5283c59a181

**Comment by ryanbourdais** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/android?version=dev:121f665a931e1653453f8b09dae65b0dd3ebf609

**Review by EricRibeiro — APPROVED** [review]
LGTM

**Pull request merged**
